alacritty_terminal/src/ansi.rs

```diff
@@ -1079,14 +1079,20 @@ where
                 handler.set_cursor_style(style);
             },
             ('r', None) => {
-                let arg0 = arg_or_default!(idx: 0, default: 1) as usize;
-                let top = Line(arg0 - 1);
                 // Bottom should be included in the range, but range end is not
-                // usually included.  One option would be to use an inclusive
+                // usually included. One option would be to use an inclusive
                 // range, but instead we just let the open range end be 1
                 // higher.
```

Comment:
This comment should stay above the other `top/bottom` block, since that's where the `-1` stuff is happening.